apps/cli/src/k8s/postgres.ts

```diff
@@ -0,0 +1,73 @@
+import { Chart, ChartProps } from "cdk8s";
+import {
+    ConfigMap,
+    IConfigMap,
+    Probe,
+    Secret,
+    Service,
+    StatefulSet,
+} from "cdk8s-plus-27";
+import { Construct } from "constructs";
+
+export interface PostgresProps extends ChartProps {
+    password: string;
+}
+
+export class Postgres extends Chart {
+    public readonly service: Service;
+    public readonly config: IConfigMap;
+    public readonly secret: Secret;
+
+    constructor(scope: Construct, id: string, props: PostgresProps) {
+        super(scope, id, props);
+
+        const labels = {
+            "app.kubernetes.io/name": "postgres",
+            "app.kubernetes.io/instance": "postgres",
+            "app.kubernetes.io/version": "13",
```

Comment:
The latest version is 16, and we already us 15 at AWS.

I suggest using 15 or 16.